common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -99,7 +100,8 @@ GSC.InPopupMainScript.prepareAndShowAsModalDialog = function() {
   GSC.InPopupMainScript.setWindowHeightToFitContent();
   GSC.InPopupMainScript.setupClosingOnEscape();
   GSC.InPopupMainScript.setupRejectionOnWindowClose();
-  GSC.InPopupMainScript.showWindow();
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP)
+    GSC.InPopupMainScript.showWindow();
```

Comment:
nit: Let's add a comment that explains why we're doing this in the App mode only. Something along the lines of:
```
// In the App packaging mode popup windows are initially opened hidden (see the flags
// passed to chrome.app.window.create() in popup-opener.js), to avoid flickering while
// the UI is being initialized. In the Extension packaging mode, there's no way to open an
// initially-hidden window, so there's no need to change the visibility here.
```